frontend/src/components/secure-message/AccessUserCard.vue

```diff
@@ -234,71 +238,133 @@ export default {
-        }).finally(() => {
-          this.$emit('refresh');
-        });
+      if(this.instituteTypeCode === 'SCHOOL'){
+        let userSchool = userToRelink.edxUserSchools.find(school => school.mincode === this.instituteCode);
+        const payload = {params:
+              {
+                userToRelink: userToRelink.edxUserID,
+                mincode: this.instituteCode,
+                userSchoolID: userSchool.edxUserSchoolID
+              }
+        };
+        ApiService.apiAxios.post(Routes.edx.EXCHANGE_RELINK_USER, payload)
+          .then(()=> {
+            this.setSuccessAlert('User has been removed, email sent with instructions to re-link.');
+          }).catch(error => {
+            this.setFailureAlert('An error occurred while re-linking a user. Please try again later.');
+            console.log(error);
+          }).finally(() => {
+            this.$emit('refresh');
+          });
+      }else{
+        let userDistrict = userToRelink.edxUserDistricts.find(district => district.districtId === this.instituteCode);
+        const payload = {params:
+              {
+                userToRelink: userToRelink.edxUserID,
+                districtId: this.instituteCode,
+                edxUserDistrictID: userDistrict.edxUserDistrictID
+              }
+        };
+        ApiService.apiAxios.post(Routes.edx.EXCHANGE_RELINK_USER, payload)
+          .then(()=> {
+            this.setSuccessAlert('User has been removed, email sent with instructions to re-link.');
+          }).catch(error => {
+            this.setFailureAlert('An error occurred while re-linking a user. Please try again later.');
+            console.log(error);
+          }).finally(() => {
+            this.$emit('refresh');
+          });
+      }
+
     },
     clickRemoveButton(userToRemove) {
-      let userSchool = userToRemove.edxUserSchools.find(school => school.mincode === this.mincode);
-      const payload = {params:
-          {
-            userToRemove: userToRemove.edxUserID,
-            mincode: this.mincode,
-            userSchoolID: userSchool.edxUserSchoolID
-          }
-      };
-      ApiService.apiAxios.post(Routes.edx.EXCHANGE_REMOVE_USER, payload)
-        .then(()=> {
-          this.setSuccessAlert('User has been removed.');
-        }).catch(error => {
-          this.setFailureAlert('An error occurred while removing a user. Please try again later.');
-          console.log(error);
-        }).finally(() => {
-          this.$emit('refresh');
-        });
+      if(this.instituteTypeCode === 'SCHOOL'){
+        let userSchool = userToRemove.edxUserSchools.find(school => school.mincode === this.instituteCode);
+        const payload = {params:
+              {
+                userToRemove: userToRemove.edxUserID,
+                mincode: this.instituteCode,
+                userSchoolID: userSchool.edxUserSchoolID
+              }
+        };
+        ApiService.apiAxios.post(Routes.edx.EXCHANGE_REMOVE_USER, payload)
```

Comment:
Same comment as above - please only use the condition for the criteria and remove the duplicate lines.